package kyrinbox.api;

message OperationLog {
    required int64 op_type = 1;
    required bytes log_data = 2;
}

message OperationLogMessage {
    optional int32 state = 1;
    required string op_id = 2;
    optional OperationLog op_log = 3;
}
